src/setup.ts

```diff
@@ -135,7 +135,7 @@ export const setup: Quest<ChronerTask> = {
           $locations`The Post-Mall, Moonshiners' Woods, The Cave Before Time, The Sleazy Back Alley`,
         );
       },
-      ready: () => AutumnAton.available(),
+      ready: () => AutumnAton.available() && !get("noncombatForcerActive"),
```

Comment:
I don't see why this is a necessary condition for that.

Reply:
Hmm, you might be right  - looking back over my logs, I think I misunderstood which action was tripping it up. will remove